debian/kafka-connect-datagen/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright 2019 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Builds a docker image for Kafka Connect
+
+FROM confluentinc/cp-kafka-connect-base
+
+MAINTAINER partner-support@confluent.io
+LABEL io.confluent.docker=true
+ARG COMMIT_ID=unknown
+LABEL io.confluent.docker.git.id=$COMMIT_ID
+ARG BUILD_NUMBER=-1
+LABEL io.confluent.docker.build.number=$BUILD_NUMBER
+ARG DATAGEN_VERSION=0.1.2
```

Comment:
Why 0.1.2?  The latest tag is `v0.1.3`

Reply:
I was going on what's listed on Confluent Hub, which is `0.1.2`

See @ewencp 's comments above re. `latest` https://github.com/confluentinc/cp-docker-images/pull/766#discussion_r307085720